Add FAQ on blocking forwarded blocked events (#7367)
segmentio/segment-docs: src/protocols/faq.md

Warehouse connectors don't use data type definitions for schema creation. The [data types](/docs/connections/storage/warehouses/schema/#data-types) for columns are inferred from the first event that comes in from the source.

> ### Can I use schema controls to block events that have been blocked from another source and forwarded to a new one?

Reviewer: ```suggestion
### Can I use schema controls to block events forwarded to my source from another source?
```

> Schema controls can only be used to block events as they are ingested into Segment. When you forward a blocked event from another source, that event has already been ingested into Segment. Thus, as it enters the second source, it bypasses the pipeline that would be used to block the event a second time. 

Reviewer: ```suggestion
You can only use schema controls to block events at the point that they are ingested into Segment. When you forward an event that Segment has previously ingested from another source, that event bypasses the pipeline that Segment uses to block events and cannot be blocked a second time. 
```
